src/platform/ESP32/ConnectivityManagerImpl_WiFi.cpp

```diff
@@ -1109,6 +1112,33 @@ void ConnectivityManagerImpl::OnStationIPv6AddressAvailable(const ip_event_got_i
 #endif
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
+static constexpr uint32_t kBeaconIntervalMs    = 100;
+static constexpr uint32_t kDefaultDTIMInterval = 3; // this is determined by the AP, use a constant value for it.
+
+CHIP_ERROR ConnectivityManagerImpl::_GetSEDIntervalsConfig(ConnectivityManager::SEDIntervalsConfig & sedIntervalsConfig)
+{
+    sedIntervalsConfig.ActiveIntervalMS = chip::System::Clock::Milliseconds32(kBeaconIntervalMs);
+#if CONFIG_WIFI_POWER_SAVE_MIN
+    sedIntervalsConfig.IdleIntervalMS = chip::System::Clock::Milliseconds32(kDefaultDTIMInterval * kBeaconIntervalMs);
+#elif CONFIG_WIFI_POWER_SAVE_MAX
+    sedIntervalsConfig.IdleIntervalMS = chip::System::Clock::Milliseconds32(CONFIG_WIFI_PS_LISTEN_INTERVAL * kBeaconIntervalMs);
+#endif
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ConnectivityManagerImpl::_SetSEDIntervalsConfig(const ConnectivityManager::SEDIntervalsConfig & intervalsConfig)
+{
+    return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE;
+}
+
+CHIP_ERROR ConnectivityManagerImpl::_RequestSEDActiveMode(bool onOff, bool delayIdle)
+{
+    return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE;
```

Comment:
Won't this mean the device can go into idle mode in the middle of an exchange, etc?  @wqx6